app/views/components/gds/link.scala.html

```diff
@@ -27,7 +27,5 @@
 
 @hasHint = @{textHidden.exists(_.nonEmpty)}
 
-<a class="@classes" href="@call" target=@target @id.map{id => id="@id" }>
- <span @if(hasHint){aria-hidden="true"}>@text</span>
- @if(hasHint){<span class="govuk-visually-hidden">@{textHidden.getOrElse("")}</span>}
-</a>
\ No newline at end of file
+<a class="@classes" href="@call" target=@target @id.map{id => id="@id" } @if(target == "_blank"){rel="noopener noreferrer"}
```

Comment:
Would be wise to add a comment here informing future devs not to split the `<a>` tag into multiple lines. The effect on messages with links is far from obvious so its likely they would not realise (and the problem will then return) :zombie:

Reply:
:muscle:

Reply:
I'll add that, but if you try to format this piece of code, then you'd have a few tests failing. So you'll definitely notice ;-)